SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -807,7 +857,8 @@ bool SpatialGDKGenerateSchemaForClasses(TSet<UClass*> Classes)
 		return false;
 	}
 
-	check(GetDefault<UGeneralProjectSettings>()->bSpatialNetworking);
+	// TODO(Alex): why is this check here?
+	//check(GetDefault<UGeneralProjectSettings>()->bSpatialNetworking);
```

Comment:
I believe this check exists so that you can't try and generate schema with a commandlet and spatial is disabled. 
If spatial is disabled schema gen gets messed up (I believe Matt has more context)

Reply:
I've put it back.